security/pkg/server/ca/server.go

```diff
@@ -137,147 +119,24 @@ func recordCertsExpiry(keyCertBundle util.KeyCertBundle) {
 	certChainExpiryTimestamp.Record(certChainExpiry)
 }
 
-// Run starts a GRPC server on the specified port.
-func (s *Server) Run() error {
-	grpcServer := s.grpcServer
-	var listener net.Listener
-	var err error
-
-	if grpcServer == nil {
-		listener, err = net.Listen("tcp", fmt.Sprintf(":%d", s.port))
-		if err != nil {
-			return fmt.Errorf("cannot listen on port %d (error: %v)", s.port, err)
-		}
-
-		var grpcOptions []grpc.ServerOption
-		grpcOptions = append(grpcOptions, s.createTLSServerOption(), grpc.UnaryInterceptor(grpc_prometheus.UnaryServerInterceptor))
-
-		grpcServer = grpc.NewServer(grpcOptions...)
-	}
+// Register registers a GRPC server on the specified port.
+func (s *Server) Register(grpcServer *grpc.Server) {
 	pb.RegisterIstioCertificateServiceServer(grpcServer, s)
-
-	grpc_prometheus.EnableHandlingTimeHistogram()
```

Comment:
I now realize the method had this side-effect on the main server. Not sure if it was ever used - but may be good to add it to server.go ( at some point in future, if telemetry believes it's needed )

Reply:
I think we still have it, if I understand correctly:
```
$ rg EnableHandlingTimeHistogram
pilot/pkg/bootstrap/server.go
200:    prometheus.EnableHandlingTimeHistogram()

security/pkg/server/ca/server.go
159:    grpc_prometheus.EnableHandlingTimeHistogram()
```